BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -263,7 +263,16 @@ extension AttributedLabel {
 
             if !isMeasuring {
                 if previousAttributedText != attributedText {
-                    links = attributedLinks(in: model.attributedText) + detectedDataLinks(in: model.attributedText)
+                    let attributedLinks = attributedLinks(in: model.attributedText)
+                    let detectedLinks = detectedDataLinks(in: model.attributedText)
+                    if let newText = attributedText {
```

Comment:
Is there a difference between `attributedText` and `model.attributedText`? IIRC one goes through `displayableAttributedText`.

Reply:
UGH `displayableAttributedText` is the whole issue here. its removing the link attribute which voiceover uses.

Reply:
Well, thats frustrating to discover. going back to draft while i recalibrate